src/containers/ProjectDashboardContainer.jsx

```diff
@@ -32,7 +32,7 @@ class ProjectDashboardContainer extends Component {
     return (
       <div>
         <h2>Project Dashboard</h2>
-        {React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
+        {project.primary_language && React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
```

Comment:
A bit of a hack, but this makes sure that child components wait for the project to load before rendering.